content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -14,22 +14,53 @@ Configuration
 Credentials tab
 ---------------
 
-Odoo needs your **API Credentials** to connect with your Stripe account, which comprises:
+The way you acquire your credentials depends on your hosting:
 
-- :ref:`Publishable Key <stripe/api_keys>`: The key solely used to identify the account with Stripe.
-- :ref:`Secret Key <stripe/api_keys>`: The key to sign the merchant account with Stripe.
-- :ref:`Webhook Signing Secret <stripe/webhook>`: When you enable your webhook on your Stripe
-  account, this signing secret must be set to authenticate the messages sent from Stripe to Odoo.
+- On Odoo Online, all you need to do is follow the onboarding.
+- On Odoo.sh and on-premise, extra steps are required.
+
+Odoo Online
+~~~~~~~~~~~
+
+#. Go to the **eCommerce** or **Sales** app and click on the *Activate Stripe* or *Set payments*
+   button on the onboarding banner.
+#. Fill in the requested information and submit the form once you are done.
+#. Confirm your email address when Stripe sends you a confirmation email.
+#. At the end of the process, you are redirected to Odoo. If you submitted all the requested
+   information, you are all set and your payment acquirer is enabled.
+
+.. tip::
+   If you prefer to use your own :ref:`API keys <stripe/api_keys>`, it is possible to do so by
+   :ref:`activating the Developer mode <developer-mode>` and :ref:`enabling Stripe manually
+   <payment_acquirers/add_new>`.
+
+On-premise or Odoo.sh
+~~~~~~~~~~~~~~~~~~~~~
+
+#. Go to the **eCommerce** or **Sales** app and click on the *Activate Stripe* or *Set payments*
+   button on the onboarding banner.
+#. Fill in the requested information and submit the form once you are done.
+#. Confirm your email address when Stripe sends you a confirmation email.
+#. At the end of the process, you are redirected to the payment acquirer **Stripe** on Odoo.
+#. :ref:`Fill in your credentials <stripe/api_keys>` and :ref:`generate a webhook <stripe/webhook>`.
+   Then, enable the payment acquirer.
 
 .. important::
-   If you are trying Stripe as a test, in the **test mode**, change the **State** to *Test
-   Mode*. We recommend doing this on a test Odoo database, rather than on your main database.
+   If you are testing Stripe (in **test mode**), change the **State** to *Test Mode*. We recommend
+   doing this on a test Odoo database rather than on your main database.
 
 .. _stripe/api_keys:
 
 Publishable and Secret keys
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Odoo needs your **API Credentials** to connect with your Stripe account, which comprises:
+
+- :ref:`Publishable Key <stripe/api_keys>`: The key solely used to identify the account with Stripe.
+- :ref:`Secret Key <stripe/api_keys>`: The key to sign the merchant account with Stripe.
+- :ref:`Webhook Signing Secret <stripe/webhook>`: When you enable your webhook on your Stripe
+  account, this signing secret must be set to authenticate the messages sent from Stripe to Odoo.
+
```

Comment:
I'd put all of this right after credentials and before line 17 (The way you acquire your credentials depends on your hosting:)

Reply:
I'm not sure it's a good idea because you don't actually need these credentials if you're on Odoo Online.

Since this PR doesn't really touch these lines (git thinks it does but it only inserts new sections between the title and the lines), I'd prefer to leave them like this so they can be referenced from both "Odoo Online" and "Odoo.sh or on-premise" sections. Then, when content tabs become available in 15.0, we'll move the tip that references this section from the "Odoo Online" section inside the appropriate tab under "Publishable and Secret keys".

Reply:
I agree it can stay afterward for now.
If they need an explanation on this, they can access this part of the page by clicking on the ref link.